lib/providers/race_events.dart

```diff
@@ -39,3 +42,71 @@ Stream<Map<String, List<int>>> teamScores(TeamScoresRef ref, String eventId) {
   });
   return controller.stream;
 }
+
+// TODO change to immutable map
```

Comment:
what about this?

Reply:
it should actually use an immutable map but it also works this way